gc/base/standard/WorkPacketsSATB.cpp

```diff
@@ -64,17 +63,10 @@ MM_WorkPacketsSATB::initialize(MM_EnvironmentBase *env)
 		return false;
 	}
 
-	if (0 == MM_GCExtensions::getExtensions(_extensions)->overflowCacheCount) {
-		/* If the user has not specified a value for overflowCacheCount

```

Comment:
where did this code move?

Reply:
For WorkPacketsSATB we decided to use OverflowStandard which doesn't use overflowCacheCount, and WorkPacketsRealtime (subclass of WorkPacketsSATB and used in metronome)  still uses IncrementalOverflow and sets overflowCacheCount (in openj9).